server/controllers/PasswordResetController.js

```diff
@@ -0,0 +1,127 @@
+import { } from 'dotenv/config';
+import bcrypt from 'bcrypt';
+import models from '../models';
+import Mailer from '../helpers/Mailer';
+import JwtHelper from '../helpers/JwtHelper';
+import emails from '../helpers/emailMessages';
+
+const { User } = models;
+
+/**
+  * This class contains all the methods responsible for user
+  * related operations on the app including creating and querying
+  * users from the database
+  * It is made up static methods which can be called from anywhere in the app.
+  */
+export default class PasswordResetController {
+  /**
+  * Checks if email exist and sends a reset email if email exit.
+  * @param {object} req the request object
+  * @param {object} res the response object
+  */
+  static sendResetEmail(req, res, next) {
+    const { email } = req.body;
+    // Jwt token
+    const token = JwtHelper.createToken({ email }, 1800);
+    // Create Reset Email Data
+    // This is to verify if user email exist and send reset
```

Comment:
Excellent work @yomigeek! Please remove these unnecessary comments. The JSdoc should contain enough information about the class/function. Also, the code should be self-explanatory.